tools/frr-reload.py

```diff
@@ -1740,10 +1740,12 @@ def compare_context_objects(newconf, running):
                 delete_bgpd = True
                 lines_to_del.append((running_ctx_keys, None))
 
-            # We cannot do 'no interface' or 'no vrf' in FRR, and so deal with it
+            elif running_ctx_keys[0].startswith("interface"):
```

Comment:
If nothing is present under `interface` context, then it's not printed at all. I'm trying to understand what problem are we solving here.

Reply:
Sorry for the confusion, I've updated the description I hope it's more clear now.

Reply:
please put the description into the commit message itself.  No-one is going to look at the actual PR in a couple of years.  They'll look at the commit via git show...

Reply:
Btw, shouldn't we check for `len(running_ctx_keys) == 1` in this case too? It might be also `interface blabla vrf ...`, right?

Reply:
It should not make a difference in my opinion.
Because now it matches everything starting with `interface` which is under the "global" configuration context.

For example it will match:
```
interface foo
```
But it will also match:
```
interface foo vrf bar
``` 
But it will not match something like:
```
some context
    interface foo
```